api/include/opentelemetry/global/provider.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include "opentelemetry/trace/noop.h"
+#include "opentelemetry/trace/tracer_provider.h"
+#include "opentelemetry/nostd/unique_ptr.h"
+
+using opentelemetry::trace::NoopTracer;
+using opentelemetry::trace::TracerProvider;
+
+namespace opentelemetry
+{
+namespace global
```

Comment:
I don't know if we want `opentelemetry::global` or just `opentelemetry`? Any opinions?

Reply:
I'm a fan of axing the `global` section, it feels redundant. Willing to put it back in if there is a compelling reason to do so.

Reply:
I agree with getting rid of the global namespace. We should also get rid of the `global/` subdir and put this stuff in `trace/` like otel-java does: https://github.com/open-telemetry/opentelemetry-java/tree/master/api/src/main/java/io/opentelemetry/trace